### ISSUE TITLE
crash on Alpine-musl

### ISSUE THREAD

**Issue by KurumiFake** [lucko/spark#205]
running `spark profiler` on Alpine-musl causes the JVM to crash
```
> spark profiler
[03:59:48 INFO]: [⚡] Initializing a new profiler, please wait...
> #
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x0000000000003f40, pid=4996, tid=5061
#
# JRE version: OpenJDK Runtime Environment (17.0.3+7) (build 17.0.3+7-alpine-r0)
# Java VM: OpenJDK 64-Bit Server VM (17.0.3+7-alpine-r0, mixed mode, sharing, tiered, compressed oops, compressed class ptrs, g1 gc, linux-aarch64)
# Problematic frame:
# C  0x0000000000003f40
#
# Core dump will be written. Default location: core.4996 (may not exist)
#
# An error report file with more information is saved as:
# /home/lazytest/hs_err_pid4996.log
#
# If you would like to submit a bug report, please visit:
#   https://gitlab.alpinelinux.org/alpine/aports/issues
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
```
happens on both aarch64 (above) and x86-64

**Comment by KurumiFake** [comment]
forgot to mention its HotSpot, not OpenJ9 or any other JVM implementations

**Comment by Experimentali** [comment]
Same issue.

**Comment by lucko** [comment]
Fixed in 06de991f44f3f0f33eed21fb92224a395a2a92ff
